pom.xml

```diff
@@ -115,11 +115,11 @@
         <artifactId>maven-war-plugin</artifactId>
         <version>2.3</version>
         <configuration>
-          <archiveClasses>true</archiveClasses>
+  	  <archiveClasses>true</archiveClasses>
```

Comment:
Ahh nothing this was just troubleshooting when I was having maven problems. Will re-add